### ISSUE TITLE
not compatible with phpBB 3.2.x

### ISSUE THREAD

**Issue by phpbb-store** [phpBBSEO/usu#114]
The current version is not compatible with phpBB 3.2.x; you can install, but clicking on tab / Extensions PCA, you get a blank page. Have you been working on this problem?

**Comment by SiteSplat** [comment]
3.2? LOL The extension is barely beta quality on 3.1.x and has many issues still. Never mind 3.2.x which many parts will need to be re-written.

**Comment by phpbb-store** [comment]
I understand. But it is the future of this development at the heart. I have struggled to update my forum seo 3.0 to 3.1 to avoid losing indexing. If the fate of this extension is to be abandoned I would like to know first. This extension is very useful and important; many forums have not yet updated waiting for an official release of this.

**Comment by lddd** [comment]
3.2? i thought 3.1.8 is latest phpBB?

**Comment by phpbb-store** [comment]
it currently is 3.1.8; the 3.2 is still in development. I refer in fact to a possible future adjustment

**Comment by Leinad4Mind** [comment]
@phpbb-store I would recommend you to convert the phpbbseo to normal phpbb. It is possible on 3.0 to revert to normal phpbb links. And after that step you can go to 3.1 and 3.2 without problems.

**Comment by phpbb-store** [comment]
Losing 5 years indexing

**Comment by Leinad4Mind** [comment]
If you convert to normal using the correct htaccess convertor, you'll not lose anything.
Read this: http://www.phpbb-seo.com/en/phpbb-seo-premod/article10171-15.html#p71442
and this: http://www.phpbb-seo.com/en/phpbb-seo-premod/article36433.html

**Comment by JamieGee123** [comment]
I have a fork of phpBB-SEO for phpBB 3.2 for personal use

**Comment by wcz-txp** [comment]
can you make it public?

**Comment by ser** [comment]
@JamieGee123 yes please make it public. It generally works for me on 3.2, but it would be cool to see your improvements.

**Comment by AroundtheGlobe** [comment]
I had a proper working and improved version for 3.1, but when I updated to 3.2 the plugin seems to be broken. 

It would be great if one phpbb seo that works for 3.2 could be made public so we can all improve it to make it work for the future.

**Comment by ser** [comment]
I have updated to 3.2 yesterday and the extension works. But of course it might depend on many factors, php version, www server, etc.

**Comment by JamieGee123** [comment]
Same story regarding 3.1, the extension worked fine but then 3.2 broke it for whatever reason. I found the errors that were thrown up in apache logs and fixed them one by one. The extension now functions correctly on the front end no problems, in admin area one of the extensions menu links is pointing to a url that ends with "htaccess". If you change that part of the url manually to "server" a .htaccess will be generated in cache folder which you can then use on your site... So yes it's not 100% working in admin but at least it will get you guys your pretty URLs back and works 100% on the front end. I'm happy to share what I have, I would also like to compare it to ser's version. I don't know how that is even working when there are core changes in phpBB 3.2 that conflict with it.

**Comment by ser** [comment]
I have a standard nginx configuration, it might be that nginx config is 3.2 proof and apache is not.

**Comment by AroundtheGlobe** [comment]
I've replaced all 3.1 files with the 3.2 ones and upgraded the forum database. From that moment the plugin stopped working. I have some problems enabling the phpbb seo plugin again because of a browser encoding error which I do not seem to be able to solve. No errors in the apache log and no way to see the real error. When I disable the language files it seems to be working again. I can see some urls are replaced in the test setup, but I can't navigate the clean urls because the route is not found. 

I am on apache and php 5.6. I will see if I can fork this plugin and include my changes and make the code public again. Any help to get the plugin working again and fix the bugs would be greatly appreciated.

**Comment by JamieGee123** [comment]
I was able to fix the language problem you are talking about using the info here https://www.phpbb.com/community/viewtopic.php?t=2349131 search through the main listener and helper  files of the extension for occurences of the "lang" and apply the changes. The third post in that thread helped if I remember rightly

**Comment by AroundtheGlobe** [comment]
@JamieGee123 would you be willing to share your code? I have been playing around with it for many days now and I get bits and pieces to work. When it works I am missing some menu options due to the language changes made in the code.

**Comment by silneil** [comment]
@AroundtheGlobe 
Click in the Jamie profile and download the fork is public.  

Thanks @JamieGee123 for this fork!

**Comment by Galixte** [comment]
His fork is exactly the same like this one.

**Comment by silneil** [comment]
So... for me work in phpbb 3.2

**Comment by AroundtheGlobe** [comment]
Makes me wonder what the difference is. @silneil is your site running on Apache?
It seems the adjustments that need to be made are in the ACP. I can get it to work, but without some important options.

**Comment by silneil** [comment]
@AroundtheGlobe  look this topic:
http://sitesplat.com/phpBB3/viewtopic.php?t=1684

**Comment by AroundtheGlobe** [comment]
Seems the information is not public and I can not signup without a specific code. :( Could you share the info  here?

**Comment by silneil** [comment]
to install you need change the name of the folder. 
root/ext/phpbbseo/usu/ (all the extensions files here)
